bubblegum/program/README.md

```diff
@@ -436,3 +436,9 @@ Decompress an NFT into an uncompressed Metaplex NFT.  This will cost rent for th
 | `data`                            | 0      | ~    | [`MetadataArgs`](https://github.com/metaplex-foundation/metaplex-program-library/blob/master/bubblegum/program/src/state/metaplex_adapter.rs#L81) object.
 
 </details>
+
+## Testing
+```bash
+cargo build
+cargo test-bpf --bpf-out-dir ../../test-programs
+```
```

Comment:
Wdyt about moving this more towards the top, i.e. after the Basic operation section but before the detailed Accounts section? Otherwise seems like it is too far down to immediately see and this is good info to have at top level.